dist/tools/flake8/check.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+#
+# Copyright (C) 2017 Alexandre Abadie <alexandre.abadie@inria.fr>
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+
+if tput colors &> /dev/null && [ $(tput colors) -ge 8 ]; then
+    CERROR="\e[1;31m"
+    CWARNING="\e[1;33m"
+    CRESET="\e[0m"
+else
+    CERROR=
+    CWARNING=
+    CRESET=
+fi
+
+RIOTBASE=$(readlink -f "$(dirname $(realpath $0))/../../..")
+
+CHECK_PATHS="${RIOTBASE}/tests \
```

Comment:
please use "changed_files.sh" as all the other static checks do

Reply:
Thanks ! That's much better indeed. I pushed an update that adds this.